woocommerce/wc-admin
# Show real orders in the sidebar orders tab

Fixes https://github.com/woocommerce/woo-dash/issues/89 - This PR creates a new ActivityList component, which uses the selected tab to show a specific list of activity. In this case, we've only built out the OrdersList view.

<img width="322" alt="screen shot 2018-06-05 at 3 27 26 pm" src="https://user-images.githubusercontent.com/541093/40998312-6b0b49e0-68d5-11e8-85c7-2fa0db988abf.png">

This does use the old design of ActivityCard, but will be updated once the design stabilizes (see #95).

This PR also adds the libraries from Calypso that handle formatting currency and calculating product taxes and totals. We'll also need to create some way of passing currency settings into the app, I'll create another PR for that.

**To Test**

- View any page on the woo-dash app
- The sidebar should have these tabs: "Orders", "Reviews", "Stock" (counts are not real)
- The orders tab should display a real list of orders on the site -- currently showing 10 most recent orders (API default)
- Other tabs should say "Coming soon…" (just to show tab-switch behavior)
- The new libraries also come with tests, so run those to make sure everything works: `npm test`
